server/routes/users.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const router = express.Router();
+const user = require('../controller/user');
+
+/* GET users listing. */
+router.get('/', (req, res, next) => {
+    res.send('respond with a resource');
+});
```

Comment:
This one looks like a left-over from the project generator :)

Reply:
Removed